Running-Mastodon/Production-guide.md

```diff
@@ -37,7 +37,9 @@ server {
   server_name example.com;
 
   ssl_protocols TLSv1.2;
-  ssl_ciphers HIGH:!MEDIUM:!LOW:!aNULL:!NULL:!SHA;
+  ssl_ciphers 'ECDHE+CHACHA20:ECDHE+AESGCM'; # or 'ECDHE:!DSS:!PSK:!SRP:!3DES:!RC4:!DES:!IDEA:!RC2:!NULL'
```

Comment:
SSL Labs isn't prioritizing AES GCM when using this cipher, but it's probably a bug, that's why I did not understand what you wanted, sorry